app/controllers/books_controller.rb

```diff
@@ -1,6 +1,21 @@
 class BooksController < ApplicationController
   def index
-    @books = Book.all
+    case params[:sort]
```

Comment:
I know this isn't required but a case statement can only take one argument at a time. Not a refactor now but something to be aware of if we take multiple queries.